UI/obs-app.cpp

```diff
@@ -2322,7 +2322,7 @@ static int run_program(fstream &logFile, int argc, char *argv[])
 
 	const char *desktop = getenv("XDG_CURRENT_DESKTOP");
 	const char *session_type = getenv("XDG_SESSION_TYPE");
-	if (session_type && desktop && strcmp(desktop, "GNOME") == 0 &&
+	if (session_type && desktop && !!strstr(desktop, "GNOME") &&
```

Comment:
There are more usage of `strstr() != NULL`  than `!!strstr`.
```suggestion
	if (session_type && desktop && strstr(desktop, "GNOME") != NULL &&
```